data/scripts.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+const {PRNG} = require('./../prng');
```

Comment:
Apparently this is not one of the things we test for.

I never said we had anywhere near 100% test coverage.

And yes, I fixed it myself in bb7b16835 - the server restart half an hour ago was because of some stray issues caused by this crash.